src/test/java/liuni/LiUniResourceTest.java

```diff
@@ -68,14 +66,16 @@ public void tearDown() {
     /* Test LiUniResource .fetchTimeline() REST method */
     @Test
     public void testREST_fetchTimeline_emptyTimeline() {
```

Comment:
for all test definitions don't use underscores

Reply:
Done